packages/utils/src/fhir-converter/README.md

```diff
@@ -0,0 +1,18 @@
+# Testing Flow
+
+Each of these scripts is used by passing in the file you want to compare:
```

Comment:
having `jsondiffpatch` is a prereq to running this right? should we add something like the following to the instructions?

# Prereqs
Run `npm install -g jsondiffpatch` to install the diffing tool

Reply:
yah. Its in the package.json but I will add to readme